src/prefect/tasks/prefect/flow_run_rename.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+from prefect import Task
+from prefect.client import Client
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class FlowRunRenameTask(Task):
```

Comment:
Would you mind changing this to `RenameFlowRunTask`? This matches our other task naming conventions, where the verb goes first (e.g. `CreateGithubPR`). The presence/absence of the `Task` suffix is a bit all over the place currently, but since this module uses `Task` suffixes that makes sense here too.

Reply:
Yeah this looks great to me, I am +1 on the name change 🙂

Reply:
Should be updated! Let me know if you want me to rename the file as well, i.e., `rename_flow_run.py`. For now I left it as is.

Reply:
Nah, that should be fine. Users import from `prefect.tasks.prefect`, so the actual file name matters less.